.github/workflows/catalogd-tilt.yaml

```diff
@@ -1,45 +0,0 @@
-name: catalogd-tilt
-
-on:
-  pull_request:
-    paths:
-      - 'catalogd/.bingo/**'
-      - 'catalogd/.github/workflows/catalogd-tilt.yaml'
-      - 'catalogd/api/**'
-      - 'catalogd/cmd/**'
-      - 'catalogd/config/**'
-      - 'catalogd/internal/**'
-      - 'catalogd/pkg/**'
-      - 'catalogd/Tiltfile'
-  merge_group:
-
-jobs:
-  tilt:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-      with:
-        fetch-depth: 0
-    - name: Clone tilt-support
-      run: |
-        git clone https://github.com/operator-framework/tilt-support catalogd/../tilt-support
-    - name: Install Go
-      uses: actions/setup-go@v5
-      with:
-        go-version-file: go.mod
-    - name: Install Tilt
-      run: |
-        TILT_VERSION="0.33.3"
-        curl -fsSL https://github.com/tilt-dev/tilt/releases/download/v$TILT_VERSION/tilt.$TILT_VERSION.linux.x86_64.tar.gz | \
-          tar -xzv -C /usr/local/bin tilt
-    - name: Install ctlptl
-      run: |
-        CTLPTL_VERSION="0.8.20"
-        curl -fsSL https://github.com/tilt-dev/ctlptl/releases/download/v$CTLPTL_VERSION/ctlptl.$CTLPTL_VERSION.linux.x86_64.tar.gz | \
-          tar -xzv -C /usr/local/bin ctlptl
-    - name: Set up kind
-      run: ctlptl create cluster kind --registry=ctlptl-registry
-    - name: Test Tilt
-      run: |
-        cd catalogd
-        tilt ci
```

Comment:
HI @joelanford 

Does that make sense if we keep it? 
Are we not now testing all by once in the tilt.yaml and Tiltfile in the root dir?

Reply:
Yeah, I think we can drop both:
- `.github/workflows/catalogd-tilt.yaml`
- `catalogd/Tiltfile`